src/api/parser/test/indexer.test.js

```diff
@@ -0,0 +1,187 @@
+const { POSTS_URL = 'http://localhost/v1/posts' } = process.env;
+const { Elastic, logger, createError } = require('@senecacdot/satellite');
+const { waitOnReady, indexPost, deletePost, search } = require('../src/utils/indexer');
+
+const index = 'posts';
+const { mock } = Elastic();
+const loggerSpy = jest.spyOn(logger, 'error').mockImplementation((msg) => msg);
+
+afterEach(() => {
```

Comment:
Can you move this inside the `describe()` below?

Reply:
Readying your tests, you can wrap the entire thing in a `describe()` and put it there.